Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim-bullseye
+
+WORKDIR fastapi-heroku
+
+COPY  . .
+
+RUN   chmod +x entrypoint.sh
```

Comment:
In order to lessen Dockerfile steps, can you make entrypoint.sh executable in the repo?

```
# On Linux/Mac
chmod +x entrypoint.sh

# On Windows
git update-index --chmod=+x entrypoint.sh
```

Reply:
@sadikkuzu Hi , I haven't worked on windows , it would be great if you can make this changes after merging , thanks

Reply:
Both commands do same thing.
Ok you can run `chmod +x entrypoint.sh` and you can delete this line `RUN chmod +x entrypoint.sh` in Dockerfile.